doc-content/enterprise-only/cases/case-management-case-file.con.adoc

```diff
@@ -22,3 +22,53 @@ You can specify any prefix. For example, if you specify the prefix `IT`, the fol
 `IT-0000000002`
 
 `IT-0000000003`
+
+You can also configure a variable as the business key, for example the following generates the `HR-TYPE1` prefix.
```

Comment:
I think it's more generic than that, shouldn't we say you can 'further customize the business key' (not just configure variable) and then document all possible options?  I'm missing quite some options from https://issues.redhat.com/browse/JBPM-9094 (unless more complete documentation is provided elsewhere?).

Reply:
Updated for next review cycle.

Reply:
Done in Design doc. This topic removed for GSG.